core/utils/utils_test.go

```diff
@@ -762,6 +763,28 @@ func TestStartStopOnce(t *testing.T) {
 	assert.Equal(t, int32(2), callsCount.Load())
 }
 
+func TestStartStopOnce_StartErrors(t *testing.T) {
+	var s utils.StartStopOnce
+
+	err := s.StartOnce("foo", func() error { return errors.New("foo") })
+	assert.Error(t, err)
+
+	var callsCount atomic.Int32
+	incCount := func() {
+		callsCount.Inc()
+	}
+
+	assert.False(t, s.IfStarted(incCount))
```

Comment:
```suggestion
	assert.False(t, s.IfStarted(callCount.Inc))
```

Reply:
This doesn't work because it returns int32